docs/quick_tutorial/authentication.rst

```diff
@@ -39,7 +39,7 @@ Steps
 
    .. literalinclude:: authentication/setup.py
     :language: python
-    :emphasize-lines: 5-6
+    :emphasize-lines: 6-7
```

Comment:
The original was in error. The preceding text says only to add ``bcrypt``. This should be ``:emphasize-lines: 7``.